.github/actions/create-timestamp/action.yml

```diff
@@ -0,0 +1,16 @@
+﻿name: create-timestamp
+author: qwerty
+description: Create a timestamp in ISO 8601 format for use in tags (replaces ":" with "." for use in git tags).
+runs:
+  using: composite
+  steps:
+    - name: Get timestamp
+      shell: pwsh
+      run: |
+        # Get the current timestamp in the ISO 8601 format but replace ":" with "." because we can't use ":" in tags
```

Comment:
```suggestion
        # Get the current timestamp in the ISO 8601 format but replace ":" with "." because we can't use ":" in tags.
```

Reply:
Please directly commit such suggestions for the reviewer doesn't have to eyeball them.